2022-03-16 21:56:39,013 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 21:56:39,015 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 21:56:39,015 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 21:57:33,229 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 21:57:33,230 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 21:57:33,230 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 21:57:46,658 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54006)
2022-03-16 21:57:46,659 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457066.658588, 'user': {'account_name': 'TestSender233333'}}
2022-03-16 21:57:47,181 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647457066.7105448, 'account_name': 'TestSender233333'}
2022-03-16 21:57:47,687 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647457067.193751, 'user': 'TestSender233333'}
2022-03-16 21:57:48,196 INFO server.py Клиент ID: 25 отключился от сервера.
2022-03-16 21:59:48,782 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54020)
2022-03-16 22:00:09,784 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457191.165157, 'user': {'account_name': 'TestSender233333'}}
2022-03-16 22:09:02,004 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54054)
2022-03-16 22:09:02,004 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457742.0044842, 'user': {'account_name': 'TestSender233333'}}
2022-03-16 22:10:06,679 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54062)
2022-03-16 22:10:06,679 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457806.678935, 'user': {'account_name': 'TestSender2333333'}}
2022-03-16 22:10:13,243 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647457806.902438, 'account_name': 'TestSender2333333'}
2022-03-16 22:10:13,757 INFO server.py Клиент ID: 26 отключился от сервера.
2022-03-16 22:10:24,364 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54064)
2022-03-16 22:10:24,365 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457824.364769, 'user': {'account_name': 'Guest'}}
2022-03-16 22:10:24,879 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647457824.412939, 'account_name': 'Guest'}
2022-03-16 22:10:25,386 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647457824.898226, 'user': 'Guest'}
2022-03-16 22:10:25,895 INFO server.py Клиент ID: 27 отключился от сервера.
2022-03-16 22:10:41,733 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54070)
2022-03-16 22:10:47,260 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647457843.870392, 'user': {'account_name': 'Guest'}}
2022-03-16 22:16:23,098 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54086)
2022-03-16 22:16:33,598 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647458183.0979111, 'user': {'account_name': 'Guest'}}
2022-03-16 22:16:44,869 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54089)
2022-03-16 22:16:44,870 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647458204.8684852, 'user': {'account_name': 'Guest'}}
2022-03-16 22:17:32,364 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 22:17:32,365 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 22:17:32,365 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 22:18:39,573 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 22:18:39,574 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 22:18:39,574 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 22:18:52,202 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54115)
2022-03-16 22:18:52,202 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647458332.201878, 'user': {'account_name': 'Guest'}}
2022-03-16 22:18:52,732 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647458332.274354, 'account_name': 'Guest'}
2022-03-16 22:18:53,238 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647458332.7527812, 'user': 'Guest'}
2022-03-16 22:18:53,755 INFO server.py Клиент ID: 27 отключился от сервера.
2022-03-16 22:19:58,661 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54120)
2022-03-16 22:19:58,662 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647458398.660814, 'user': {'account_name': 'Guest'}}
2022-03-16 22:20:28,739 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54126)
2022-03-16 22:20:38,415 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647458430.212081, 'user': {'account_name': 'Guest'}}
2022-03-16 22:28:53,954 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 22:28:53,957 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 22:28:53,958 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 22:31:30,431 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54179)
2022-03-16 22:32:14,933 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647459104.160537, 'user': {'account_name': 'dsjfghsdgafjhg'}}
2022-03-16 22:32:26,779 INFO server.py Клиент ID: 28 отключился от сервера.
2022-03-16 22:32:48,051 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54183)
2022-03-16 22:32:48,556 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647459168.051678, 'user': {'account_name': 'dwqdqwd'}}
2022-03-16 22:32:49,080 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647459168.6121411, 'account_name': 'dwqdqwd'}
2022-03-16 22:32:49,588 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647459169.1002948, 'user': None}
2022-03-16 22:32:49,590 INFO server.py Клиент ID: 29 отключился от сервера.
2022-03-16 22:38:58,585 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-16 22:38:58,586 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-16 22:38:58,586 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-16 22:39:21,142 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54210)
2022-03-16 22:39:21,150 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647459561.1418061, 'user': {'account_name': 'dasdas'}}
2022-03-16 22:39:21,673 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647459561.207753, 'account_name': 'dasdas'}
2022-03-16 22:39:22,179 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647459561.689643, 'user': None}
2022-03-16 22:39:22,182 INFO server.py Клиент ID: 1 отключился от сервера.
2022-03-16 22:46:32,381 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54245)
2022-03-16 22:46:32,382 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647459992.381823, 'user': {'account_name': 'апаафыпаыв'}}
2022-03-16 22:46:32,899 INFO server.py Клиент ID: 2 отключился от сервера.
2022-03-16 22:55:00,322 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54279)
2022-03-16 22:55:00,330 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647460500.322196, 'user': {'account_name': 'dsafdsgfdsag'}}
2022-03-16 22:55:00,847 INFO server.py Клиент ID: 3 отключился от сервера.
2022-03-17 00:16:17,845 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 00:16:17,853 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 00:16:17,853 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 00:16:32,074 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 00:16:32,075 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 00:16:32,075 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 00:25:37,878 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 00:25:37,879 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 00:25:37,880 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 00:29:53,861 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54618)
2022-03-17 00:30:04,362 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466193.861595, 'user': {'account_name': 'TestSender1'}}
2022-03-17 00:30:05,776 INFO server.py Клиент ID: 4 отключился от сервера.
2022-03-17 00:30:11,739 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54619)
2022-03-17 00:30:11,740 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466211.739908, 'user': {'account_name': 'TestSender1'}}
2022-03-17 00:30:24,229 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54621)
2022-03-17 00:30:24,732 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466224.2300029, 'user': {'account_name': 'TestSender12'}}
2022-03-17 00:30:25,255 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466224.752471, 'account_name': 'TestSender12'}
2022-03-17 00:30:25,763 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466225.26631, 'user': 'TestSender12'}
2022-03-17 00:30:34,313 INFO server.py Клиент ID: 5 отключился от сервера.
2022-03-17 00:31:27,749 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54624)
2022-03-17 00:31:27,834 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466287.749177, 'user': {'account_name': 'TestSender12'}}
2022-03-17 00:31:37,061 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54625)
2022-03-17 00:31:41,741 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466297.0620642, 'user': {'account_name': 'TestSender122'}}
2022-03-17 00:31:42,263 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466301.764322, 'account_name': 'TestSender122'}
2022-03-17 00:31:42,768 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466302.280061, 'user': 'TestSender122'}
2022-03-17 00:31:47,305 INFO server.py Клиент ID: 6 отключился от сервера.
2022-03-17 00:32:47,371 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54628)
2022-03-17 00:32:47,372 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466367.370991, 'user': {'account_name': 'TestSender1222'}}
2022-03-17 00:32:52,832 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466367.5541701, 'account_name': 'TestSender1222'}
2022-03-17 00:33:28,169 INFO server.py Клиент ID: 7 отключился от сервера.
2022-03-17 00:33:30,252 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54631)
2022-03-17 00:33:30,753 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466410.2525508, 'user': {'account_name': 'TestSender1222'}}
2022-03-17 00:33:41,627 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54633)
2022-03-17 00:33:42,129 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466421.627627, 'user': {'account_name': 'TestSender1222'}}
2022-03-17 00:33:51,159 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54635)
2022-03-17 00:33:51,663 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466431.159992, 'user': {'account_name': 'TestSender12222'}}
2022-03-17 00:33:52,188 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466431.68506, 'account_name': 'TestSender12222'}
2022-03-17 00:33:52,693 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466432.204772, 'user': 'TestSender12222'}
2022-03-17 00:34:09,806 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647466449.7022521, 'user': 'TestSender12222', 'account_name': 'апаафыпаыв'}
2022-03-17 00:34:23,409 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647466463.13234, 'user': 'TestSender12222', 'account_name': 'TestSender1222'}
2022-03-17 00:36:14,378 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54643)
2022-03-17 00:36:14,549 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466574.37886, 'user': {'account_name': 'TestSender_gui'}}
2022-03-17 00:36:16,636 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466574.563243, 'account_name': 'TestSender_gui'}
2022-03-17 00:36:17,146 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466576.65572, 'user': 'TestSender_gui'}
2022-03-17 00:36:28,721 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647466588.22109, 'user': 'TestSender_gui', 'account_name': 'TestSender12222'}
2022-03-17 00:36:35,777 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender12222', 'time': 1647466595.044447, 'mess_text': 'dcvfaeftwrtb'}
2022-03-17 00:36:35,778 INFO server.py Отправлено сообщение пользователю TestSender12222 от пользователя TestSender_gui.
2022-03-17 00:36:41,314 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647466601.0232792, 'user': 'TestSender12222', 'account_name': 'TestSender_gui'}
2022-03-17 00:36:49,396 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender12222', 'to': 'TestSender_gui', 'time': 1647466609.0001762, 'mess_text': 'dvbrtwbrtwbe'}
2022-03-17 00:36:49,396 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender12222.
2022-03-17 00:37:04,215 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender12222', 'time': 1647466619.200633, 'mess_text': 'dabertbwretb'}
2022-03-17 00:37:04,223 INFO server.py Отправлено сообщение пользователю TestSender12222 от пользователя TestSender_gui.
2022-03-17 00:37:32,272 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender12222', 'to': 'TestSender_gui', 'time': 1647466647.257813, 'mess_text': 'vdfvdfvdfv'}
2022-03-17 00:37:32,272 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender12222.
2022-03-17 00:37:44,070 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender12222', 'to': 'TestSender1222', 'time': 1647466659.055184, 'mess_text': 'dfv d '}
2022-03-17 00:37:44,071 INFO server.py Связь с клиентом с именем TestSender1222 была потеряна
2022-03-17 00:39:56,607 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 00:39:56,608 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 00:39:56,608 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 00:40:14,419 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54659)
2022-03-17 00:40:14,419 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466814.4191082, 'user': {'account_name': 'TestSender12222'}}
2022-03-17 00:40:14,940 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466814.436388, 'account_name': 'TestSender12222'}
2022-03-17 00:40:15,447 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466814.958626, 'user': 'TestSender12222'}
2022-03-17 00:40:25,259 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54661)
2022-03-17 00:40:25,764 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647466825.2594502, 'user': {'account_name': 'TestSender_gui'}}
2022-03-17 00:40:26,285 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647466825.780848, 'account_name': 'TestSender_gui'}
2022-03-17 00:40:26,792 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647466826.3039842, 'user': 'TestSender_gui'}
2022-03-17 00:40:34,349 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender12222', 'time': 1647466833.893153, 'mess_text': 'gfdgdfgdf'}
2022-03-17 00:40:34,350 INFO server.py Отправлено сообщение пользователю TestSender12222 от пользователя TestSender_gui.
2022-03-17 00:40:42,908 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender12222', 'to': 'TestSender_gui', 'time': 1647466842.8285718, 'mess_text': 'dfgdfgdfg'}
2022-03-17 00:40:42,908 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender12222.
2022-03-17 00:41:02,530 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender12222', 'to': 'TestSender_gui', 'time': 1647466862.173368, 'mess_text': 'dgdfgdfg'}
2022-03-17 00:41:02,531 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender12222.
2022-03-17 00:41:10,082 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647466870.070456, 'account_name': 'TestSender12222'}
2022-03-17 00:41:11,604 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647466871.465191, 'account_name': 'TestSender_gui'}
2022-03-17 00:44:10,255 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54674)
2022-03-17 00:44:20,755 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647467050.255168, 'user': {'account_name': 'TestSender1'}}
2022-03-17 00:44:26,638 INFO server.py Клиент ID: 4 отключился от сервера.
2022-03-17 00:45:13,555 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 00:45:13,556 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 00:45:13,556 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 00:45:27,287 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54677)
2022-03-17 00:45:27,287 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647467127.28718, 'user': {'account_name': 'TestSender1'}}
2022-03-17 00:45:27,806 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467127.3046799, 'account_name': 'TestSender1'}
2022-03-17 00:45:28,314 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647467127.823903, 'user': 'TestSender1'}
2022-03-17 00:45:50,036 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 54679)
2022-03-17 00:45:50,538 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647467150.036288, 'user': {'account_name': 'TestSender_gui'}}
2022-03-17 00:45:51,062 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467150.5585861, 'account_name': 'TestSender_gui'}
2022-03-17 00:45:51,565 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647467151.080069, 'user': 'TestSender_gui'}
2022-03-17 00:46:01,133 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender12222', 'time': 1647467160.4120471, 'mess_text': 'csdcsdsd'}
2022-03-17 00:46:01,142 ERROR server.py Пользователь TestSender12222 не зарегистрирован на сервере, отправка сообщения невозможна.
2022-03-17 00:46:12,712 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647467172.3626418, 'user': 'TestSender_gui', 'account_name': 'TestSender1'}
2022-03-17 00:46:19,273 DEBUG server.py Разбор сообщения от клиента : {'action': 'remove', 'time': 1647467179.172651, 'user': 'TestSender_gui', 'account_name': 'TestSender1'}
2022-03-17 00:46:26,819 DEBUG server.py Разбор сообщения от клиента : {'action': 'remove', 'time': 1647467186.402584, 'user': 'TestSender_gui', 'account_name': 'TestSender12222'}
2022-03-17 00:46:32,369 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647467191.9630988, 'user': 'TestSender_gui', 'account_name': 'TestSender1'}
2022-03-17 00:46:41,931 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender1', 'time': 1647467201.653455, 'mess_text': 'dcsdscs'}
2022-03-17 00:46:41,939 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender_gui.
2022-03-17 00:46:44,459 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647467204.018292, 'user': 'TestSender1', 'account_name': 'TestSender_gui'}
2022-03-17 00:46:59,059 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender1', 'to': 'TestSender_gui', 'time': 1647467218.910423, 'mess_text': 'sdfdsfsd'}
2022-03-17 00:46:59,059 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender1.
2022-03-17 00:47:08,122 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender1', 'to': 'TestSender_gui', 'time': 1647467227.902721, 'mess_text': 'fsdsdfsdf'}
2022-03-17 00:47:08,123 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender1.
2022-03-17 00:47:14,663 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender1', 'time': 1647467234.6410408, 'mess_text': 'sdfsdfsd'}
2022-03-17 00:47:14,663 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender_gui.
2022-03-17 00:47:47,745 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender1', 'to': 'TestSender122', 'time': 1647467263.354035, 'mess_text': 'sdfsdf'}
2022-03-17 00:47:47,746 ERROR server.py Пользователь TestSender122 не зарегистрирован на сервере, отправка сообщения невозможна.
2022-03-17 00:48:19,056 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647467294.037815, 'user': 'TestSender_gui', 'account_name': 'TestSender12222'}
2022-03-17 00:48:25,312 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467303.05351, 'account_name': 'TestSender_gui'}
2022-03-17 00:48:26,087 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467305.3462431, 'account_name': 'TestSender_gui'}
2022-03-17 00:48:29,107 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467308.687623, 'account_name': 'TestSender_gui'}
2022-03-17 00:48:32,638 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647467312.5093591, 'account_name': 'TestSender_gui'}
2022-03-17 00:48:40,185 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647467320.134897, 'user': 'TestSender_gui', 'account_name': 'TestSender12222'}
2022-03-17 00:48:45,722 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender12222', 'time': 1647467325.51106, 'mess_text': 'sdfsdf'}
2022-03-17 00:48:45,722 ERROR server.py Пользователь TestSender12222 не зарегистрирован на сервере, отправка сообщения невозможна.
2022-03-17 00:49:11,858 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647467351.565768, 'account_name': 'TestSender_gui'}
2022-03-17 00:49:15,892 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647467355.109322, 'account_name': 'TestSender1'}
2022-03-17 13:27:21,860 DEBUG loging_decos.py Запущено приложение ServerApp,c параметрами ['/Users/dmitry/Desktop/Programms/geekbrains/11-PyQT/PyQT_pycharm/Lesson_5/server.py', 'server.py', '-p', '8888', '-a', '192.168.0.50'] функции listen_port).
2022-03-17 13:27:21,862 INFO server.py Запущен сервер, порт для подключений: 8888, адрес с которого принимаются подключения: 192.168.0.50. Если адрес не указан, принимаются соединения с любых адресов.
2022-03-17 13:27:21,862 DEBUG server.py Server launched. IP: 192.168.0.50 Port: 8888
2022-03-17 13:30:35,711 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 55159)
2022-03-17 13:30:35,850 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647513035.7114592, 'user': {'account_name': 'TestSender1'}}
2022-03-17 13:30:40,559 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647513035.891986, 'account_name': 'TestSender1'}
2022-03-17 13:30:41,078 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647513040.585278, 'user': 'TestSender1'}
2022-03-17 13:30:55,680 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647513055.430093, 'account_name': 'TestSender1'}
2022-03-17 13:32:35,091 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 55167)
2022-03-17 13:32:35,092 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647513155.091638, 'user': {'account_name': 'TestSender1'}}
2022-03-17 13:32:35,598 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647513155.098421, 'account_name': 'TestSender1'}
2022-03-17 13:32:36,107 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647513155.618525, 'user': 'TestSender1'}
2022-03-17 13:32:49,190 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647513168.7657752, 'account_name': 'TestSender1'}
2022-03-17 13:33:52,930 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 55170)
2022-03-17 13:33:52,931 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647513232.930717, 'user': {'account_name': 'TestSender1'}}
2022-03-17 13:33:53,446 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647513232.943817, 'account_name': 'TestSender1'}
2022-03-17 13:33:53,951 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647513233.463609, 'user': 'TestSender1'}
2022-03-17 13:34:51,883 INFO server.py Установлено соедение с Клиентом: ('192.168.0.50', 55173)
2022-03-17 13:34:52,001 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647513291.883391, 'user': {'account_name': 'TestSender_gui'}}
2022-03-17 13:34:56,992 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647513292.0076559, 'account_name': 'TestSender_gui'}
2022-03-17 13:34:57,503 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647513297.0208378, 'user': 'TestSender_gui'}
2022-03-17 13:35:05,554 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender1', 'time': 1647513305.464951, 'mess_text': 'dfgdsfagdsf'}
2022-03-17 13:35:05,555 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender_gui.
2022-03-17 13:35:09,075 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender1', 'to': 'TestSender_gui', 'time': 1647513308.757985, 'mess_text': 'dsfasdf'}
2022-03-17 13:35:09,075 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender1.
2022-03-17 13:35:19,631 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender1', 'to': 'TestSender_gui', 'time': 1647513319.52393, 'mess_text': 'sdfsdf'}
2022-03-17 13:35:19,631 INFO server.py Отправлено сообщение пользователю TestSender_gui от пользователя TestSender1.
2022-03-17 13:36:18,756 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'TestSender_gui', 'to': 'TestSender1', 'time': 1647513375.7186182, 'mess_text': 'sdfsdfsd'}
2022-03-17 13:36:18,756 INFO server.py Отправлено сообщение пользователю TestSender1 от пользователя TestSender_gui.
2022-03-17 13:36:46,511 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647513405.861699, 'account_name': 'TestSender1'}
2022-03-17 13:36:53,134 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647513412.661172, 'account_name': 'TestSender_gui'}
